script/inspect_schemas.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+"""Inspect all component SCHEMAS."""
```

Comment:
This script helped to identify how to handle CONFIG_SCHEMA etc. It also shows where we don't have any Schemas and where we have conflicting schemas (lie `zigbee`). I only fixed script in this PR